src/index.ts

```diff
@@ -107,12 +92,19 @@ function updateCropArea(event: TouchEvent | MouseEvent | KeyboardEvent) {
   if (deltaX && deltaY) updateDimensions(el, deltaX, deltaY, !(event instanceof KeyboardEvent))
 }
 
+function getShadowHost(el: HTMLElement) {
+  const rootNode = el.getRootNode()
+  if (!(rootNode instanceof ShadowRoot)) return el
+  return rootNode.host
+}
```

Comment:
It would be good if we could avoid this function, but I haven't been able to find an alternative that allows me to access the Shadow DOM host. Open to ideas; otherwise, we can keep this as-is for now.

Reply:
You could inline it with something like `el.getRootNode()?.host ?? el`